runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsOverlay.java

```diff
@@ -232,6 +231,43 @@ private void renderPlayerOverlay(Graphics2D graphics, Player actor, Color color)
 					textLocation.getY());
 			}
 		}
+
+		if (plugin.isShowAgilityLevel())
+		{
+			if (!plugin.agilityMap.isEmpty() && plugin.agilityMap.keySet().contains(actor.getName()))
+			{
+				if (plugin.getAgilityFormat().equals(PlayerIndicatorsPlugin.AgilityFormats.ICONS))
+				{
+					int width = graphics.getFontMetrics().stringWidth(name);
+					int height = graphics.getFontMetrics().getHeight();
+					if (plugin.agilityMap.get(actor.getName()) >= 70)
```

Comment:
`70` should be a config option or a `static final` field

Reply:
Done